### ISSUE TITLE
v.voronoi.skeleton missing in newer qgis where grass8 is used

### ISSUE THREAD

**Issue by stefankinell** [qgis/QGIS#56365]
### What is the bug or the crash?

In earlier qgis installations where grass7 was used there was function called v.voronoi.skeleton. This function seems to be missing in later installations I have, e.g. 3.43.3, which uses grass8. In my case I have a model that uses v.voronoi.skeleton that I built earlier that now refuse to run due to lack of v.voronoi.skeleton.

### Steps to reproduce the issue

1Try running a previous model that is using v.voronoi.skeleton.
2. pop up dialoge says "can not run the algorithm, the model you try to run contains an algorithm that is not available: grass7:v.voronoi.skeleton.

### Versions

QGIS-version
3.34.3-Prizren
QGIS kodrevision
47373234ac
Qt version
5.15.3
Python version
3.9.18
GDAL/OGR version
3.8.3
PROJ version
9.3.1
EPSG Registry database version
v10.098 (2023-11-24)
GEOS version
3.12.1-CAPI-1.18.1
SQLite version
3.41.1
PDAL version
2.6.0
PostgreSQL client version
15.2
SpatiaLite version
5.1.0
QWT version
6.1.6
QScintilla2 version
2.13.4
Version av OS
Windows 10 Version 2009

Aktiva python plugin
becagis
23.7.19
d3datavis
3.0.6
FreehandRasterGeoreferencer
0.8.3
geometric_attributes
0.4
GeometryShapes
0.7
HCMGIS
24.1.12
LAStools
2.0.1
layertilesmapcanvas
1.4
MBTiles2img
0.4
nominatim
1.4.5
qgis-plugin-findreplace-main
1
Qgis2threejs
2.7.2
qtiles
1.7.2
raster_divider
1.3
SelectWithin
0.4
SRTM-Downloader
3.2.1
ThRasE
23.8
db_manager
0.1.20
MetaSearch
0.3.6
processing
2.12.99

### Supported QGIS version

- [X] I'm running a supported QGIS version according to [the roadmap](https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule).

### New profile

- [ ] I tried with a new [QGIS profile](https://docs.qgis.org/latest/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles)

**Comment by agiudiceandrea** [comment]
@stefankinell, thanks for reporting.
Does any other GRASS-GIS processing algorithm work in the model? Is the `v.voronoi.skeleton` GRASS-GIS processing algorithm listed in the Processing Toolbox panel? Does it work if launched from the Processing Toolbox? Does the `processing.algorithmHelp("grass7:v.voronoi.skeleton")` command outputs the algorithm help if executed from the Python console?

**Comment by stefankinell** [comment]
I do not know what went wrong for me but after diging more today I got it to work.

I did not have GRASS in the processing toolbox. Neither on my Windows installation of QGIS 3.34.3 or Mac OS 3.30. I then looked in the pluing setting and the links to the modules and plugins did not seem to be right. They seemed to lead to non existing folders.

My resolution was simply; download and install grass GIS again. restart QGIS. And then the GRASS-functions are available in the processing toolbox, and the model can be opened and run.

What puzzles me is that I can not recall having mixtured with the plugin settings. And I have never gotten this to work after grass 8 being installed with later qgis-versions. Or might it be me installing something else that messed the connection up. If so, I must have done it both on my Mac and Windows-computer.

If no one else has experienced similar issues we blame it on human errors.
